plugins/modules/vmware_tag_manager.py

```diff
@@ -178,6 +178,9 @@ def __init__(self, module):
         if self.object_type == 'Datacenter':
             self.managed_object = self.pyv.find_datacenter_by_name(self.object_name)
 
+        if self.object_type == 'Datastore':
```

Comment:
While you are here, would you be interested in adding the remaining object types like

- Folder
- Datastore Cluster
- Resource Pool

advTHANKSance

Reply:
@Akasurde happy to take a look.  i have some other stuff to take care of this week, so it may take a few days for me to get around to it, but happy to help!

Reply:
@Akasurde Should have all of those objects done and have tested in my own local vSphere environment.  I split them up into separate commits.  Please let me know if you have any questions.